docsite/docusaurus.config.ts

```diff
@@ -86,6 +92,15 @@ const config: Config = {
             },
           ],
         },
+        {
```

Comment:
this adds another footer section titled Community with a single broken link to a page that doesn't exist. What are you trying to achieve with this?